kaldb/src/main/java/com/slack/kaldb/metadata/cache/CacheSlotMetadata.java

```diff
@@ -4,6 +4,7 @@
 import com.slack.kaldb.proto.metadata.Metadata;
 
 public class CacheSlotMetadata extends KaldbMetadata {
+  public static final String METADATA_SLOT_NAME = "SLOT";
```

Comment:
Seems like an unused variable?

Reply:
Yeah, it's just due to how I split this off of the previous PR. Both of the cache slot string variables will be used in the `ChunkManager` refactor. I put them into this PR so the `ChunkManager` refactor has to touch fewer classes, as that's gonna be a pretty big update.